### DISCUSSION TITLE
RabbitMQ Streams Capabilities

### DISCUSSION THREAD

**Discussion by Raguckas86** [rabbitmq/rabbitmq-dotnet-client#1072] [Q&A]
All of us are very excited about streams. Before jumping in projects and starting using it would be nice to have one place where we can discuss on RabbitMQ Streams Capabilities and if it can meet our project requirements. 

Please be free to share any requirement for streaming and let's have a discussion is it capabilities to meet it. 

**I can start with the simple one:**

1. Are RabbitMQ streams capable to route Log/Event data to like "Kafka Topics" based on message data/metadata? and Is there any function like "kafka topic based access control?

**Comment by lukebakken** [comment]
Please see the documentation:

https://www.rabbitmq.com/docs/streams
